### DISCUSSION TITLE
Predict the performance of wind turbines

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1761] [Q&A]
Can OPENFAST predict the performance of wind turbines the same as WT_Perf? 
If it can do this, how to use the OPENFAST to predict the performance of wind turbines?

**Comment by jjonkman** [comment]
Dear @sjihf12,

What do you mean when you say, "the same as WT_Perf"?  If you are asking if OpenFAST can be used to generate aerodynamic-only performance maps (e.g., Cp/Ct/Cq dependent on TSR and pitch), then answer is "yes".  The standalone driver for AeroDyn can be used generate aerodynamics-only performance curves.  Moreover, if you run AeroDyn coupled with structures and control within OpenFAST, you can generate performance maps including the effects of aero-elasticity.

Best regards,

**Reply by sjihf12** [reply]
Thank you!
